grama/eval_pnd.py

```diff
@@ -52,18 +54,17 @@ def eval_pnd(model, df_train, df_test, sign, n=int(1e4), seed=None):
     >>>...
     >>> md = (
     >>>     df_train
-    >>>     >> gr.ft_foo()
+    >>>     >> gr.ft_gdp()
     >>> )
-    >>> ( pred_model
+    >>>
+    >>> pnd_results = (
+            pred_model
             >> gr.ev_pnd(
-                model,
                 df_train,
                 df_test,
-                signs = {
-                    'y1':-1,
-                    'y2':1
-                },
-                seed = 101
+                signs = {"y1":1, "y2":1},
+                seed = 101,
+                append = True
             )
```

Comment:
A nice addition to the example would be to sort the DataFrame by the resulting `pr_score` column. This highlights how this code would be used in practice: To rank a set of candidates

Reply:
i.e. `gr.tf_arrange(gr.desc(DF.pr_scores))`